packages/shared/src/components/onboarding/OnboardingHeader.tsx

```diff
@@ -60,44 +63,49 @@ export const OnboardingHeader = ({
 
   if (activeScreen !== OnboardingStep.Intro) {
     return (
-      <header className="sticky top-0 z-3 mb-10 flex w-full justify-center backdrop-blur-sm">
-        <img
-          className="pointer-events-none absolute left-0 right-0 top-0 z-0 max-h-[12.5rem] w-full"
-          src={getImage()}
-          alt="Gradient background"
-        />
-        <div className="flex w-full max-w-4xl items-center justify-between !px-4 py-10 tablet:!px-6">
-          <ConditionalWrapper
-            condition={showPlusIcon}
-            wrapper={(component) => (
-              <div className="flex flex-row items-center gap-1">
-                {component}
-                <PlusUser
-                  iconSize={IconSize.Small}
-                  typographyType={TypographyType.Title3}
+      <>
+        {activeScreen === OnboardingStep.Plus && isFreeTrialExperiment && (
+          <PlusFreeTrialAlert />
+        )}
+        <header className="sticky top-0 z-3 mb-10 flex w-full justify-center backdrop-blur-sm">
```

Comment:
Just checking it can only show on onboarding page?
It doesn't show the other sticky alert bar there?

Can you just verify locally as they might overlap each other.

Reply:
Where I can find the other sticky bar? Couldn't find some example 👀